Formula/mtoc.rb

```diff
@@ -3,6 +3,7 @@ class Mtoc < Formula
   homepage "https://opensource.apple.com/"
   url "https://github.com/apple-oss-distributions/cctools/archive/refs/tags/cctools-949.0.1.tar.gz"
   sha256 "8b2d8dc371a57e42852fa6102efaf324ef004adf86072bf9957e2ac9005326c1"
+  revision 1
```

Comment:
No need to force everyone to reinstall
```suggestion
```

Reply:
There is need, because our products will no longer build correctly, and this is essentially a new version.

Reply:
In that case I'm going to insist we wait for an actual new version.

Reply:
There is no new version, modern cctools do not build with mtoc, likely because Apple no longer supports this due to migrating to ARM, so we are the maintainers essentially.

Reply:
In that case it should be deprecated, not monkey patched.

Reply:
I do not think so, because the tool is the only thing that allows to build UEFI applications on macOS. There are many tools out in the world that are monkey patched as you call it, even as popular as unzip.

I am personally fine Homebrew removes mtoc from its formulae, though, to me it feels a rather dumb approach, but leaving something broken is not good for us, because we will be bugged about why X does not work as it happened before.